deebot_client/messages/json/pos.py

```diff
@@ -0,0 +1,46 @@
+"""Position messages."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from deebot_client.events import Position, PositionsEvent, PositionType
+from deebot_client.message import HandlingResult, MessageBodyDataDict
+
+if TYPE_CHECKING:
+    from deebot_client.event_bus import EventBus
+
+
+class OnPos(MessageBodyDataDict):
+    """On pos message."""
+
+    name = "onPos"
+
+    @classmethod
+    def _handle_body_data_dict(
+        cls, event_bus: EventBus, data: dict[str, Any]
+    ) -> HandlingResult:
+        """Handle message->body->data and notify the correct event subscribers.
+
+        :return: A message response
+        """
+        positions = []
+
+        for type_str in ["deebotPos", "chargePos"]:
```

Comment:
Are we sure that we are getting multiple positions on a `onPos` message? I think it's only possible on the `getPos` command and `onPos` is only providing a single position.